index.js

```diff
@@ -1,5 +1,47 @@
 function scoreCalculator(arr) {
-  // do work here
+  // not an array
+  if (!Array.isArray(arr)) {
+    throw new Error('Not an array!');
+  }
+  // not every element of array is a number
+  if (
+    !arr.every(function(elem) {
```

Comment:
It's best to do the `.every` outside of an if and place the result into a variable, then test the variable in the if.